developerportal/templatetags/app_tags.py

```diff
@@ -124,22 +126,27 @@ def pagination_additional_filter_params(request):
     """
 
     QUERY_PARAMS_TO_PASS_ON = (
-        # PAGINATION_QUERYSTRING_KEY,  # we DON'T want the page param
+        # PAGINATION_QUERYSTRING_KEY,  # we DON'T want this one: it's added separately
         TOPIC_QUERYSTRING_KEY,
         ROLE_QUERYSTRING_KEY,
         COUNTRY_QUERYSTRING_KEY,
         DATE_PARAMS_QUERYSTRING_KEY,
+        SEARCH_QUERYSTRING_KEY,
     )
 
     output_params_strings = []
 
     input_keys = request.GET.keys()
+
     for input_key in input_keys:
         if input_key not in QUERY_PARAMS_TO_PASS_ON:
             continue
         input_params = request.GET.getlist(input_key)
         for input_param in input_params:
-            output_params_strings.append(f"{input_key}={input_param}")
+            # Note that using `urllib.parse.quote` avoids spaces appearing in
+            # the output, which breaks Safari (because the space triggers a web
+            # search, not a HTTP request to the site)
```

Comment:
This was fun to discover:
![Screenshot 2020-06-01 at 14 40 44](https://user-images.githubusercontent.com/101457/83429834-93002d00-a42c-11ea-8410-9606ae5b65cc.png)